js/languages/en-US.json

```diff
@@ -21,7 +21,8 @@
   },
   "connectedPeersPage": {
     "heading": "Connected Peers",
-    "infoMessage": "Please note: If any peer does not have their profile set, clicking on their guid below will result in a User Not Found page.",
+    "totalPeers": "%{totalPeers} Found",
+    "infoMessage": "If you do not see any peers below, you are not connected to the OpenBazaar network. If a peer does not have information available on the network, clicking on their guid below will result in a User Not Found page.",
```

Comment:
Aren't there cases where users can connect to their server, but their server can't see any other nodes?

I know this happens in v1.